src/commands/executeFunction.ts

```diff
@@ -73,7 +73,7 @@ export async function executeFunction(context: IActionContext, node?: FunctionTr
             'Content-Type': 'application/json',
         });
         if (client) {
-            headers['x-functions-key'] = (await client.listHostKeys()).masterKey;
+            headers.set('x-functions-key', (await client.listHostKeys()).masterKey ?? '');
```

Comment:
This is the same command that I was having issues with while testing log streaming btw. I have a feeling something is going on with that API and it's down again (regarding the customer issue)

Reply:
I was able to repro and verify that this fixed it.

Reply:
What authority was your function though? If it was anonymous, you don't need a master key, but I think you would if it's function or admin.